hack/deployer/runner/ocp3.go

```diff
@@ -180,7 +180,7 @@ func (d Ocp3Driver) runAnsibleDockerContainer(action string) error {
 		-e USER={{.User}} \
 		-e USER_HOME={{.HomeVolumeMountPath}} \
 		-v {{.HomeVolumeName}}:{{.HomeVolumeMountPath}} \
-		-e CLOUDSDK_CONFIG={{.GCloudSDKPath}} \
+		-e CLOUDSDK_CONFIG \
```

Comment:
I am not sure I understand why this is needed.

Reply:
This is to pass that env var to the container if it's set.

Reply:
I hesitate to remove `CLOUDSDK_CONFIG` completely as it seems useful to me mainly when we want to run a CI thing without Docker and without altering our existing gcloud config dir. When we run the CI container the gcloud config dir lives in the named Docker volume.

If we keep the variable, I think it would be good to add a comment to explain what this is for to avoid any ambiguity. Something along these lines:
```go
// Propagate the variable that could have been set to not alter an existing gcloud config directory, 
// in the case of a local test outside docker
```

Reply:
Let's add a comment as Thibault suggested here?